data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
```

Comment:
`A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space`
This isn't really correct. Postverta and co. aren't bordering on Korath space.
```suggestion
	description "A damaged Kas'lor Ik 582 is stuck in a newly accessible system, near Postverta. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
```